modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/RowVersionFreeList.java

```diff
@@ -83,7 +80,7 @@ public RowVersionFreeList(
                 LOG,
                 metaPageId,
                 initNew,
-                pageListCacheLimit,
+                null,
```

Comment:
If this thing doesn't exist anymore then why do we have this "null" argument?

Reply:
Fix it